### PR TITLE
Add Spring Boot Data JPA part

### PR DESCRIPTION
**Pull request by OlesiaKapustynska**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,9 +64,14 @@
 			<artifactId>spring-orm</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-dbcp2</artifactId>
-			<version>2.11.0</version>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<version>${liquibase.version}</version>
```
Comment (yevheniisynytsia) [inline pom.xml:69]: do you have `${liquibase.version}` defined in the `<properties>` block?

File: src/main/java/com/example/bookstore/repository/BookRepository.java [hunk 2]
```diff
@@ -3,8 +3,11 @@
 import com.example.bookstore.model.Book;
 import java.util.List;
 import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
-public interface BookRepository {
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
     Book save(Book book);
```
Comment (yevheniisynytsia) [inline src/main/java/com/example/bookstore/repository/BookRepository.java]: please remove custom methods, they will be accessible by default after extending `JpaRepository`

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (yevheniisynytsia) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml]:
```suggestion
                    primaryKey: true
```
primaryKey is nonnull by default

File: src/main/java/com/example/bookstore/repository/BookRepository.java [hunk 4]
```diff
@@ -3,8 +3,11 @@
 import com.example.bookstore.model.Book;
 import java.util.List;
 import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
-public interface BookRepository {
+@Repository
```
Comment (yevheniisynytsia) [inline src/main/java/com/example/bookstore/repository/BookRepository.java]: tip: `@Repository` is not necessary if you extend `JpaRepository`, but you may leave it if you want

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Great job! Left some minor comments to improve the solution

File: pom.xml (hunk 1 above)
Comment (OlesiaKapustynska) [reply pom.xml:69]: in spring-boot-dependencies

**Review by yevheniisynytsia — APPROVED** [review]
🔥

**Pull request merged**